openedx/core/djangoapps/programs/management/commands/backpopulate_program_credentials.py

```diff
@@ -39,20 +40,17 @@ def add_arguments(self, parser):
         )
 
     def handle(self, *args, **options):
-        programs_config = ProgramsApiConfig.current()
-        self.client = Client.objects.get(name=programs_config.OAUTH2_CLIENT_NAME)
-
-        if self.client.user is None:
-            msg = (
-                'No user is associated with the {} OAuth2 client. '
-                'A service user is necessary to make requests to the Programs API. '
-                'No tasks have been enqueued. '
-                'Associate a user with the client and try again.'
-            ).format(programs_config.OAUTH2_CLIENT_NAME)
-
-            raise CommandError(msg)
+        catalog_config = CatalogIntegration.current()
+
+        try:
+            user = User.objects.get(username=catalog_config.service_username)
+        except:
+            raise CommandError(
+                '''User with username [{}] not found.
```

Comment:
I used the multi-line string to avoid the pep8 line length limit. Can you clarify the change you're suggesting?